samples/nrf9160/modem_shell/src/location/location_srv_ext_nrf_cloud.c

```diff
@@ -10,7 +10,6 @@
 #include <net/nrf_cloud_pgps.h>
 #include <net/nrf_cloud_location.h>
 #include <modem/location.h>
-#include "nrf_cloud_codec.h"
```

Comment:
Should an include be added for "nrf_cloud_defs.h" here?

Reply:
its included from nrf_cloud.h